pkg/k8s/endpoints.go

```diff
@@ -31,9 +33,15 @@ import (
 // frontend ports of the corresponding service.
```

Comment:
While you're here, can you add a comment indicating that this is developed from either an Endpoints or EndpointSlices object? Just so it's clear that we prefer EndpointSlices and fall back to Endpoints for very old versions.

Reply:
Added comment.